stack/loader.go

```diff
@@ -44,45 +44,16 @@ func NewLoader(root string) Loader {
 // Load loads a stack from dir directory. If the stack was previously loaded, it
 // returns the cached one.
 func (l Loader) Load(dir string) (S, error) {
-	logger := log.With().
-		Str("action", "Load()").
-		Str("stack", dir).
-		Logger()
-
-	logger.Trace().
-		Msg("Get relative path to root directory.")
-	stackpath := project.RelPath(l.root, dir)
-	if s, ok := l.stacks[stackpath]; ok {
-		return s, nil
-	}
-
-	fname := filepath.Join(dir, config.Filename)
-
-	logger.Debug().
-		Str("configFile", fname).
-		Msg("Parse config file.")
-	cfg, err := hcl.ParseFile(fname)
+	stack, found, err := l.TryLoad(dir)
```

Comment:
Wondering if instead of TryLoad/Load we could just have Load and a NotExist error, like os.IsNotExist. But unrelated to the PR, just sharing.